modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -185,6 +185,92 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "Verify it's possible to add an object entry with the trigger On Order Status Update"
+	@priority = "4"
+	test CanCreateActionAddObjectEntryWithOnOrderStatusUpdate {
+		task ("Given an action in the Commerce Order object") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
```

Comment:
Use the testcase ticket to complement the object name.

Ex.
```
ObjectAdmin.addObjectViaAPI(
        labelName = "Custom Object 171861",
	objectName = "CustomObject171861",
	pluralLabelName = "Custom Objects 171861");
```

Remember to replace all object names